test/helpers/fixture_set.rb

```diff
@@ -29,8 +29,10 @@ def app_name
 
     def refute_resource_exists(type, name, beta: false)
       client = beta ? v1beta1_kubeclient : kubeclient
-      client.public_send("get_#{type}", name, namespace) # 404s
-      flunk "#{type} #{name} unexpectedly existed"
+      res = client.public_send("get_#{type}", name, namespace)
+      if res.metadata.deletionTimestamp.nil? || res.metadata.deletionTimestamp == ""
```

Comment:
I think you can use `.blank?` in this project. Why was this change needed?

Reply:
When testing for deletion, some objects on 1.11 were marked for deletion but weren't deleted when we ran this check. If the deletion timestamp is checked, k8s garbage collector will clean up the resources so it made sense to check this as well in the case the res wasn't deleted